live-build/config/hooks/82-upgrade-repository.binary

```diff
@@ -118,6 +117,7 @@ fi
 sed -i "s/@@DEPENDS@@/$DEPENDS/" delphix-entire.ctl
 sed -i "s/@@VARIANT@@/$APPLIANCE_VARIANT/" delphix-entire.ctl
 sed -i "s/@@VERSION@@/$VERSION/" delphix-entire.ctl
```

Comment:
I haven't really looked at this change yet, but.. for customer images (with the virtualization package installed), the version should be the same (since they'll all still use the virtualization package's version, and that package will be the same in all the artifacts), right?

Reply:
Yeah you are right. I didn't recall we added that logic there, although I remember we discussed about it. You can drop that comment then.

Reply:
Yeah, the version will be the same for all variants that have virtualization installed (not just the customer images). I figured it probably doesn't matter too much for the other variants.